verification/curator-service/ui/src/components/new-case-form-fields/Events.tsx

```diff
@@ -4,17 +4,99 @@ import { KeyboardDatePicker } from 'formik-material-ui-pickers';
 import { MuiPickersUtilsProvider } from '@material-ui/pickers';
 import React from 'react';
 import Scroll from 'react-scroll';
+import { WithStyles } from '@material-ui/core/styles/withStyles';
+import { createStyles } from '@material-ui/core/styles';
+import { withStyles } from '@material-ui/core';
 
-export default class Events extends React.Component<{}, {}> {
+const styles = () =>
+    createStyles({
+        fieldRow: {
+            marginBottom: '2em',
+        },
+        field: {
+            width: '50%',
+        },
+    });
+
+type Props = WithStyles<typeof styles>;
+class Events extends React.Component<Props, {}> {
     render(): JSX.Element {
+        const { classes } = this.props;
         return (
             <Scroll.Element name="events">
                 <fieldset>
                     <legend>Events</legend>
                     <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                        <div className={classes.fieldRow}>
+                            <Field
+                                className={classes.field}
+                                name="confirmedDate"
+                                label="Confirmed case date"
+                                format="yyyy/MM/dd"
+                                maxDate={new Date()}
+                                minDate={new Date('2019/12/01')}
+                                component={KeyboardDatePicker}
+                            />
+                        </div>
+                        <div className={classes.fieldRow}>
```

Comment:
worth having some a util to generate these fields, since they're very similar? i'm OK with keeping as is too

Reply:
Yea good call, done.